src/HW/client/src/components/ListControls/ListControls.css

```diff
@@ -4,7 +4,7 @@
 .list-controls{
   display: flex
 }
-.taskInput {
+#taskInput {
```

Comment:
better use classes in css to control the style of other elements that might look the same
you can still keep the id and the class name